### DISCUSSION TITLE
OpenTX 1.8.0 : How to bind....?

### DISCUSSION THREAD

**Discussion by Dronard** [OpenI6X/opentx#298] [Q&A]
![20221109_144539](https://user-images.githubusercontent.com/98915844/200867076-2d51d953-b545-4b92-8a10-05ce124fd58d.jpg)
![20221109_144546](https://user-images.githubusercontent.com/98915844/200867094-0ef5df8d-97ac-4dd0-bd16-b8075fb24767.jpg)

![20221109_144652](https://user-images.githubusercontent.com/98915844/200867103-ed5e5a03-e569-4b5f-8b30-1a200f83d7af.jpg)

Here is the menu : I want to set AFHDS2 Receiver, there's a problem, no?

**Comment by ajjjjjjjj** [comment]
Hi, scroll down to line where you see `[Bnd]` and focus it using BIND button, then press OK.

**Comment by psicho100pro** [comment]
I recommend it, there is very useful information about the capabilities of opentx: http://rcdiy.ca/

**Comment by Dronard** [comment]
oh, i'm wrong i'm wrong, I hope there will be a new firmware which will get afhds compatibility, in addition to afhds2

**Reply by psicho100pro** [reply]
marianomd is working on it, but he's a busy man #168

**Reply by Dronard** [reply]
okay
